docs/bugs.json

```diff
@@ -1,4 +1,15 @@
 [
+    {
+        "uid": "SOL-2022-3",
+        "name": "DataLocationChange",
+        "summary": "It was possible to override a function and changing the data location of the parameters or return variables from ``calldata`` to ``memory`` and vice-versa. This caused invalid code to be generated when calling such a function internally.",
+        "description": "TODO",
+        "link": "TODO",
```

Comment:
Those still need to be filled in I guess - apart from that the fix generally looks good.